README.md

```diff
@@ -1,4 +1,20 @@
 # TFExplorer
+[![Build Status](https://travis-ci.org/hardlyhuman/TFExplorer.svg?branch=master)](https://travis-ci.org/hardlyhuman/TFExplorer)
+
+
+<b>TFExplorer enables developers to quickly find out the relevant TensorFlow API's for their needs & requirements.
+It basically deals with different tools for TensorFlow API's .
+This repository is for Backend Development part &  Django is the tech stack we are using for it.</b>
```

Comment:
Instead do something like:

This is the repository for backend code

The tech stack is:
   - django : <version>
   - python : <version>

SO that we can add more things later